en/micro-integrator/docs/references/connectors/salesforcebulk-connector/salesforcebulk-connector-configuration.md

```diff
@@ -0,0 +1,36 @@
+# Setting up the SalesforceBulk Environment  
+
+The SalesforceBulk connector allows you to access the [SalesforceBulk REST API](https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/) through WSO2 ESB. 
```

Comment:
We need to mention for which usecases users need to select SF bulk connector.

Reply:
Updated.